File: app/src/main/java/org/lemonadestand/btb/features/login/models/Values.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Values {
    @SerializedName("description")
    Description description;

    @SerializedName("items")
    List<Items> items;


    public void setDescription(Description description) {
        this.description = description;
    }
    public Description getDescription() {
        return description;
    }

    public void setItems(List<Items> items) {
        this.items = items;
    }
    public List<Items> getItems() {
        return items;
    }

}

File: app/src/main/java/org/lemonadestand/btb/core/models/Values.kt
package org.lemonadestand.btb.core.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Values(
	@SerializedName("description")
	var description: Description? = null,

	@SerializedName("items")
	var items: List<Items>? = null,
) : Parcelable
